CONTRIBUTING.md

```diff
@@ -36,11 +36,11 @@ We recommend to run "npx grunt eslint" before uploading a pull request.
 ## Contributing code
 
 * **Use your own fork of TAU** repository in order to prepare pull request (PR).
-* If you prepare PR which is **not ready for review** yet, please add **"in progress"** label
-* If your PR is **ready for review**, plase add **"in review"** label
-* Assign TAU Maintainers for each PR which is ready to review
+* If you prepare PR which is **not ready for review** yet, please use **"Convert to draft"** github feature
```

Comment:
We can add a link how to use this feature:

[Convert to draft](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request#creating-the-pull-request)